R/pkg/R/mllib.R

```diff
@@ -195,9 +198,9 @@ print.summary.GeneralizedLinearRegressionModel <- function(x, ...) {
     "Number of Fisher Scoring iterations: ", x$iter, "\n", sep = "")
   cat("\n")
   invisible(x)
-  }
+}
 
-#' Make predictions from a generalized linear model
+#' Makes predictions from a fitted MLlib model
```

Comment:
This line appeared in the ```Description``` section of each ```rdname```, so ```predict``` for all models will share this description and it should be general.